core/web/pipeline_job_spec_errors_controller_test.go

```diff
@@ -22,17 +22,29 @@ func TestPipelineJobSpecErrorsController_Delete_2(t *testing.T) {
 	// FindJob -> find error
 	j, err := app.JobORM().FindJob(testutils.Context(t), jID)
 	require.NoError(t, err)
-	require.Len(t, j.JobSpecErrors, 2)
-	jse := j.JobSpecErrors[1]
-
-	resp, cleanup := client.Delete(fmt.Sprintf("/v2/pipeline/job_spec_errors/%v", jse.ID))
+	t.Log(j.JobSpecErrors)
```

Comment:
nit: do we want to keep the log line?

Reply:
Back when we were spamming everything to stdout this was definitely a problem, but the test logs are nicely encapsulated